src/main/java/com/opentok/util/HttpClient.java

```diff
@@ -412,7 +416,83 @@ public String forceDisconnect(String sessionId, String connectionId) throws   Op
 
         return responseString;
     }
+    public String sipDial(String sessionId, String token, SipProperties props) throws OpenTokException {
```

Comment:
@JayTokBox can we also change this to `dial` to keep it consistent with the API method?